CHANGELOG.md

```diff
@@ -103,6 +103,7 @@ Bug fixes:
 - Fixed an issue where `Dict.partition (always (always True/False)) dict` would not be reported
 - Fixed an issue where `List.filterMap f [ a, Nothing, b ]` would be fixed to `List.filterMap f [ a, b ]`
 - Fixed an issue where `Random.map << always` would be fixed to `Random.constant`
+- Fixed an issue where `List.any (\\z -> x == y) list` would be fixed to `List.member (f x) list`
```

Comment:
Where did the `f` come from in this example?

Reply:
Corrected in one of the next commits